### PR TITLE
Fix undefined sync status reporting in tray icon

### PR DESCRIPTION
**Pull request by claucambra**
Also includes some nice clean ups

### CODE REVIEW

File: src/gui/folderman.cpp [hunk 1]
```diff
@@ -1545,54 +1545,66 @@ void FolderMan::trayOverallStatus(const QList<Folder *> &folders,
...
+
+            if (folderResult.hasUnresolvedConflicts()) {
                 *unresolvedConflicts = true;
+            }
         }
-        if (errorsSeen > 0) {
+
+        if (errorsSeen) {
             *status = SyncResult::Error;
         } else if (abortOrPausedSeen > 0 && abortOrPausedSeen == cnt) {
```
Comment (mgallien) [inline src/gui/folderman.cpp:1599]: `abortOrPausedSeen` is now handled like a boolean in code so this comparison does not make sense

**Review by mgallien — CHANGES_REQUESTED** [review]

File: src/gui/folderman.cpp (hunk 1 above)
Comment (claucambra) [reply src/gui/folderman.cpp]: okay, fixed

**Review by mgallien — APPROVED** [review]

**Pull request merged**